images/base/files/usr/local/bin/entrypoint

```diff
@@ -180,6 +180,10 @@ enable_network_magic(){
   cp /etc/resolv.conf /etc/resolv.conf.original
   sed -e "s/${docker_embedded_dns_ip}/${docker_host_ip}/g" /etc/resolv.conf.original >/etc/resolv.conf
 
+  # use dns to resolve hostnames with more priority than files
+  cp /etc/nsswitch.conf /etc/nsswitch.conf.original
```

Comment:
what was the original value?
why aren't we statically rewriting this in the baseimage build?

Reply:
debian *should* be `hosts: dns [!UNAVAIL=return] files` I think, which is actually a little surprising, I thought most distros were shipping ~`files dns`

https://github.com/golang/go/issues/35305
https://manpages.debian.org/stretch/manpages/nsswitch.conf.5.en.html

Reply:
You are correct, the original was **files dns**

Reply:
I didn't think about the original bare honestly, added to the entry point because is where we are adding these hacks